README.md

```diff
@@ -470,5 +470,7 @@ Rocket.Chat will be free forever, but you can help us speed-up the development!
 
 [![Bitcoins](https://github.com/RocketChat/Rocket.Chat.Docs/blob/master/1.%20Contributing/Donating/coinbase.png?raw=true)](https://www.coinbase.com/checkouts/ac2fa967efca7f6fc1201d46bdccb875)
 
+LIVE from MEXICO!
```

Comment:
Can you add a header above which says "Countries We've Presented At"? :)

Reply:
@graywolf336 I've added the header, what do you think?